src/test/java/bio/terra/service/dataset/DatasetDaoTest.java

```diff
@@ -834,4 +835,39 @@ public void columnOrderCreationTest() throws Exception {
     datasetDao.delete(dataset1Id);
     datasetDao.delete(dataset2Id);
   }
+
+  @Test
+  public void patchDatasetPhsId() throws Exception {
+    UUID datasetId = createDataset("dataset-create-test.json");
+
+    assertThat(
+        "dataset's PHS ID is null before patch",
+        datasetDao.retrieve(datasetId).getPhsId(),
+        equalTo(null));
+
+    String phsIdSet = "phs000000";
+    DatasetPatchRequestModel patchRequestSet = new DatasetPatchRequestModel().phsId(phsIdSet);
+    datasetDao.patch(datasetId, patchRequestSet);
+    assertThat(
+        "dataset's PHS ID is set from patch",
+        datasetDao.retrieve(datasetId).getPhsId(),
+        equalTo(phsIdSet));
+
+    String phsIdOverride = "phs111111";
+    DatasetPatchRequestModel patchRequestOverride =
+        new DatasetPatchRequestModel().phsId(phsIdOverride);
+    datasetDao.patch(datasetId, patchRequestOverride);
+    assertThat(
+        "dataset's PHS ID is overridden from patch",
+        datasetDao.retrieve(datasetId).getPhsId(),
+        equalTo(phsIdOverride));
+
+    datasetDao.patch(datasetId, new DatasetPatchRequestModel());
+    assertThat(
```

Comment:
Could we add another case here where the PHS ID is "unset" by changing it to an empty string?

Reply:
Added!